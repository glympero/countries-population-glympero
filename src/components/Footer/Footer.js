import React from "react";
import { Link } from "react-router-dom";

const Footer = () => (
  <footer>
    <div className="container">
      <div className="footer-menu">
        <Link to="/">Home</Link>
        <Link to="/chart">Chart</Link>
      </div>
    </div>
  </footer>
);

export default Footer;
