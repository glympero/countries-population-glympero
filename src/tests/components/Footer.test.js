import React from "react";
import { shallow } from "enzyme";
import Footer from "../../components/Footer/Footer";

test("should render footer correctly", () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper.find("Link").length).toBe(2);
  expect(wrapper.find("Link").first().prop("to")).toBe("/");
  expect(wrapper.find("Link").at(1).prop("to")).toBe("/chart");

  expect(wrapper).toMatchSnapshot();
});
