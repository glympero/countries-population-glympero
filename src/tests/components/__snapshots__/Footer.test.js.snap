// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render footer correctly 1`] = `ShallowWrapper {}`;
